res/openvpn-gui-res-no.rc

```diff
@@ -187,9 +187,9 @@ BEGIN
     LTEXT "OpenVPN GUI v" PACKAGE_VERSION " - Ett Windows-Grensesnitt mot OpenVPN\n" \
           "Copyright (C) 2004-2005 Mathias Sundman <info@openvpn.se>\n" \
           "http://openvpn.se/", 0, 36, 15, 206, 26
-    LTEXT "OpenVPN - En applikasjon for sikker overføring av IP nett " \
+    LTEXT "OpenVPN - En applikasjon for sikker overføring av IP-nett " \
```

Comment:
This comes from the following text in the copyright notice in openvpn codebase and an equivalent text is present in all language files. So leave as is until the default About text is changed?
 ```
 * OpenVPN -- An application to securely tunnel IP networks
 *             over a single TCP/UDP port, with support for SSL/TLS-based
 *             session authentication and key exchange,
 *             packet encryption, packet authentication, and
 *             packet compression.
```